README.md

```diff
@@ -1 +1,56 @@
-# mailserver
\ No newline at end of file
+# mailserver
+
+This server is supposed to handle TCP connections to receive emails from clients.
```

Comment:
Did we not say that we can also send a e-mail with smpt to another server or was this an optional use case?

This server is supposed to handle TCP connections to receive emails with IMAP and send emails with SMTP to another server.